.github/ISSUE_TEMPLATE/CONTENT.yml

```diff
@@ -0,0 +1,39 @@
+name: "Content Report 📄"
+description: Report an issue, a possible bug or anything that is confusing with Tauri docs.
+labels: ["improve documentation"]
+body:
+  - type: markdown
+    attributes:
+      value: "# Reminder: If you're reporting a minor correction, feel free to submit a PR directly!"
+  - type: input
+    id: subject
+    attributes:
+      label: 📚 Subject area/topic
+      description: Documentation area/topic (e.g. Authenticator, File System, CLI)
+      placeholder: "..."
+    validations:
+      required: true
+  - type: textarea
+    id: page
+    attributes:
+      label: "📋 Page(s) affected (or suggested, for new content)"
+      description: Please provide the URL of the page(s) affected (or suggested, for new content).
+      placeholder: https://tauri.app/v1/guides/getting-started/setup
+    validations:
+        required: true
+  - type: textarea
+    id: general_desc
+    attributes:
+      label: "📋 General description or bullet points (if proposing new content)"
+      description: If you are proposing new content please provide a general description or bullet points about what should be added.
+      placeholder: "..."
+    validations:
+        required: true
+  - type: textarea
+    id: incorrect_content
+    attributes:
+      label: "🖥️ Reproduction in StackBlitz (if reporting incorrect content or code samples)"
+      description: If you are reporting incorrect content or code samples please attach a reproduction.
+      placeholder: "..."
+    validations:
+        required: false
```

Comment:
I don't think we need this one 🤔 I've never come across a scenario where we've _needed_ a repro for docs in Tauri. Screenshots are a lot more useful for me for someone to annotate what they're referring to if needed.